src/index.ts

```diff
@@ -1,12 +1,13 @@
-import express from 'express'
+require('dotenv').config();
+import express from 'express';
 
-const app = express()
-const port = 3000
+const app = express();
+const port = process.env.PORT;
 
 app.get('/', (_req, res) => {
-  res.send('Hello! I am event maker telegram bot')
-})
+    res.send('Hello! I am event maker telegram bot');
```

Comment:
I though we would use 2 spaces and no semicolon

Reply:
Discussed, will be changed if it's inconvenient.